package.json

```diff
@@ -13,7 +13,10 @@
     "check-types": "tsc --noEmit --skipLibCheck",
     "clean": "rm -rf .artifact-output .yext/ dist/ localData sites-rendered-output",
     "generate": "plop",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "smartling-scan": "i18next",
+    "smartling-push": "node i18n/index.js push",
+    "smartling-pull": "node i18n/index.js pull"
```

Comment:
This is very forward-looking, but in case we ever depart from Smartling, can we make the names of these commands more generic? Something like `i18n-*` or `translations-*`

Reply:
yeah, done